docs/development.md

```diff
@@ -9,78 +9,77 @@
 To create a local copy of Presidio repository, follow [Github instructions](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
 on how to clone a project using git.
 The project is structured so that:
-- Each Preisio service has a designated directory. In it, the source code for:
-    - The service logic.
-    - Tests, both unit and integration.
-    - Serving it as an HTTP service (found in app.py).
-    - Python Packaging setup script (setup.py).
-- In the project root directory, you will find common code for using, serving and testing Presidio
-as a cluster of services, as well as CI/CD pipelines codebase and documentation.
+
+-   Each Preisio service has a designated directory. In it, the source code for:
```

Comment:
Would replace this to:
Each Presidio service has a designated directory containing: